npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -165,10 +166,22 @@ export async function getTempDir (): Promise<string> {
 }
 
 export async function getAngularCliModules (projectRoot: string) {
+  let angularVersion: string
```

Comment:
The code looks reasonable. I'd like to see an End to End testing verifying Angular 16.1 is working - can you add one here? https://github.com/cypress-io/cypress/blob/develop/npm/webpack-dev-server/cypress/e2e/angular.cy.ts#L5

Just follow the existing patterns, you can add a new Angular 16.1 project. Thanks!

Reply:
I added the E2E project for Angular 16.1. It seems you'll need to kick off the pipeline again.

Reply:
Doing this now.